### ISSUE TITLE
Import private key for did web

### ISSUE THREAD

**Issue by nitro-neal** [TBD54566975/ssi-service#542]
Be able to pull in the private key for issuing creds on existing cred:
example: https://www.tbd.website/.well-known/did.json

**Comment by decentralgabe** [comment]
I don't think we should promote moving private keys. It's safer to generate a new key in the service and have it added to an existing DID doc.

**Comment by nitro-neal** [comment]
Ok so to be clear, we wont support importing a private key? People bringing their own key wont be supported basically?

I was hoping we could get this feature so we can import our key for our did web

**Comment by decentralgabe** [comment]
> Ok so to be clear, we wont support importing a private key? People bringing their own key wont be supported basically?
> 
> 
> 
> I was hoping we could get this feature so we can import our key for our did web

I'd recommend no. To support DID web let's add a new key to the doc that's managed by the service. So we can support an "import existing DID" flow.

**Comment by andresuribe87** [comment]
I agree that we shouldn't be moving private keys.

Can you elaborate on what the "import existing DID" flow would do?

**Comment by decentralgabe** [comment]
1. Be able to import an existing DID Doc from an external source or copy/paste
2. Be able to generate a new key locally and add it to the DID Doc (at least in the `verificationMethods` block), where the new key has a controller property set as the kid of the original key (or none!?)
3. Return the updated DID doc

Then you could have a key in the service that can act as the DID without having all keys in the document
